masschallenge/impact-api
# [AC-7638] Show masquerade button on profile page

## [AC-7638](https://masschallenge.atlassian.net/browse/AC-7638)

**Testing**
- checkout to this branch on front-end, impact and accelerate
  - on accelerate: `$make run-all-servers`
  - on front-end: `$yarn start`
- As a staff user, go to expert profile page
- Confirm this PR meets the definition of done according to [AC-7638](https://masschallenge.atlassian.net/browse/AC-7638)

[accelerate PR](https://github.com/masschallenge/accelerate/pull/2745)
[front-end PR](https://github.com/masschallenge/front-end/pull/255)
